### PR TITLE
Convert DSL to markdown part 2/3

### PR DESCRIPTION
**Pull request by Hamms**
Part of the process of removing HTML from DSL fields:

1. Enable support for markdown in relevant DSL fields (https://github.com/code-dot-org/code-dot-org/pull/32750)
2. Update all DSL levels to use markdown rather than HTML
   1. levels whose HTML can be directly converted to markdown (links, bold, italic, etc) (https://github.com/code-dot-org/code-dot-org/pull/32761)
   2. levels whose HTML is already close to markdown (manually-styled code blocks, etc) **<- this PR**
   3. levels that will need to be manually refactored
3. Disable HTML

This PR updates several levels whose use of HTML can be approximately converted to markdown. Most of these updates result in minor visual changes. Some examples are below.

Level Name | Screenshot | Description
--- | --- | ---
Commute Assess - comp 2 - block | Screenshot from 2020-01-21 16-11-21 | "op4" in question updated to use code block instead of just monospace font
Commute Assess - comp 4 - block | Screenshot from 2020-01-21 16-15-58 | "true", "false", and "list" in question and answers updated to use code block instead of just monospace font
Commute Assess - cond 3 - block | Screenshot from 2020-01-21 16-28-15 | answers using `pre` tags updated to use `code` tags

### Verification Links

Following is a list of all levels updated by this PR, and a link to that level on an adhoc. The list can be used to verify that the visual changes are acceptable.

Level Name | Link
--- | ---
Commute Assess - comp 2 - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15901
Commute Assess - comp 2 - block_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15902
Commute Assess - comp 2 - block_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15903
Commute Assess - comp 2 - block_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15904
Commute Assess - comp 4 - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15906
Commute Assess - comp 4 - block_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15907
Commute Assess - comp 4 - block_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15908
Commute Assess - comp 4 - block_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15909
Commute Assess - cond 3 - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15914
Commute Assess - cond 3 - block_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15915
Commute Assess - cond 3 - block_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15916
Commute Assess - cond 3 - block_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15917
Commute Assess - func 1 - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15918
Commute Assess - func 2a - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15919
Commute Assess - func 2a - block_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15920
Commute Assess - func 2a - block_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15921
Commute Assess - func 2a - block B | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15922
Commute Assess - func 2a - block_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15923
Commute Assess - func 3a - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15929
Commute Assess - func 3a - block_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15930
Commute Assess - func 3a - block_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15931
Commute Assess - func 3a - block_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15932
Commute Assess - func 3b - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15933
Commute Assess - func 3b - block_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15934
Commute Assess - func 3b - block_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15935
Commute Assess - func 3b - block_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15936
commute assess - loops 3 - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15937
commute assess - loops 3 - block_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15938
commute assess - loops 3 - block_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15939
commute assess - loops 3 - block_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15940
CommuteAssess - Cond 1 - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15942
CommuteAssess - Cond 1 - block_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15943
CommuteAssess - Cond 1 - block_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15944
CommuteAssess - Cond 1 - block B | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15945
CommuteAssess - Cond 1 - block_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15946
CommuteAssess - Cond 2 - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15947
CommuteAssess - Cond 2 - block_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15948
CommuteAssess - Cond 2 - block_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15949
CommuteAssess - Cond 2 - block B | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15950
CommuteAssess - Cond 2 - block_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15951
CommuteAssess - Cond 4a - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15952
CommuteAssess - Cond 4a - block_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15953
CommuteAssess - Cond 4a - block_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15954
CommuteAssess - Cond 4a - block_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15955
CommuteAssess - loops 2 - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15962
CommuteAssess - loops 2 - block_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15963
CommuteAssess - loops 2 - block_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15964
CommuteAssess - loops 2 - block_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15965
CommuteAssess - loops 5 - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15966
CommuteAssess - loops 5 - block_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15967
CommuteAssess - loops 5 - block_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15968
CommuteAssess - loops 5 - block_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15969
CommuteAssess - Var 5 - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/15978
CSD U6 test events multi | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/16440
csp-u1-exam1-binary-numeric-value_2020 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/16929
csp-u2-exam1-cutting-paths_2020 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/16945
csp-u2-exam1-fault-tolerance_2020 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/16949
csp-u2-exam1-redundancy_2020 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/16956
csp-u2-exam1-routing_2020 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/16957
csp-u3-exam1-debug3_2020 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/16964
csp-u3-exam1-debug4_2020 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/16965
csp-u3-exam1-debug5_2020 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/16966
csp-u5-exam1-list-iteration-2_2020 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/16990
csp-u5-exam1-list-iteration-3_2020 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/16991
csp-u5-exam1-simulation-2_2020 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/16996
CSPFinal_Question_13 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17077
CSPFinal_Question_28 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17111
CSPFinal_Question_30 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17118
cspu5_assess3_bakeSale_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17392
cspu5_assess3_bakeSale_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17393
cspu5_assess3_bakeSale_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17394
CumulAssess - Var 2 - block | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17487
CumulAssess - Var 2 - block_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17488
CumulAssess - Var 2 - block_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17489
CumulAssess - Var 2 - block B | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17490
CumulAssess - Var 2 - block_exam_prep | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17491
U2L9 What is digital divide MC | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17892
U2L9 What is digital divide MC_2018 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17893
U2L9 What is digital divide MC_2018_2019 | http://adhoc-markdownify-stage-two-studio.cdn-code.org/levels/17894

### CODE REVIEW

**Review by davidsbailey — APPROVED** [review]

File: dashboard/config/scripts/commute_assess_comp_2_block.multi [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name 'Commute Assess - comp 2 - block'
-question 'The procedure <span style="font-family: monospace">op4</span> takes in 3 numbers, what does this procedure do?'
+editor_experiment ''
```
Comment (dmcavoy) [inline dashboard/config/scripts/commute_assess_comp_2_block.multi:2]: Probably not an issue but wanted to call out this was added on this level but not the ones below which are the same level just for different years

**Review by dmcavoy — APPROVED** [review]
I don't see links to view the 2 encrypted levels which were the ones I was going to check visually but as long as you have done that I think this is good to go

File: dashboard/config/scripts/commute_assess_comp_2_block.multi (hunk 1 above)
Comment (davidsbailey) [reply dashboard/config/scripts/commute_assess_comp_2_block.multi:2]: this looks like my bug -- editor_experiment shouldn't be getting added when the value is nil/empty. not sure what could be causing the difference though

Comment (Hamms) [reply dashboard/config/scripts/commute_assess_comp_2_block.multi:2]: the difference is just that I edited the first one in my local levelbuilder interface so I could verify the results, then copied the changes into the other files using my text editor

**Comment by Hamms** [comment]
Oh, weird; I'm not sure why those got skipped when I generated the list. But I did indeed check them manually myself, and they're fine.

**Pull request merged**